test/ejabberd_tests/tests/connect_SUITE.erl

```diff
@@ -116,6 +119,11 @@ init_per_group(feature_order, Config) ->
     config_ejabberd_node_tls(Config, fun mk_value_for_compression_config_pattern/0),
     ejabberd_node_utils:restart_application(ejabberd),
     Config;
+init_per_group(dh_tls, Config) ->
```

Comment:
There is probably no need for dedicated `dh_tls` group. I'd rather add the `dhfile` parameter to default MongooseIM settings.